### DISCUSSION TITLE
Contribution - translations

### DISCUSSION THREAD

**Discussion by voyy** [cypress-io/cypress#9130] [Questions and Help] [answered]
Hello, I have a question regarding documentation translation. Is it on hold because of cypress-io/cypress-documentation#3152 ? I wanted to contribute and do the polish (PL) translation, but I don't know if the documentation https://github.com/cypress-io/cypress-documentation/blob/master/CONTRIBUTING.md#translating is up to date right now.

**Comment by jennifer-shehane** [comment, marked as answer]
We've removed translations and put other translation of the docs on hold for now for a few reasons. 

- With our current framework, the number of translated doc files were taking a prohibitively long time to build, so that it was delaying releases.
- We're in the middle of transitioning the docs to a new framework.
- Looking at the pageviews of our docs, we found that most languages were not getting high enough pageviews for us to warrant maintaining (aside from maybe Chinese).
- There were some other issues with the current translations - https://github.com/cypress-io/cypress-documentation/issues/1686. Mainly that they were not kept in sync with the English docs, which we think offers a pretty poor experience.

Maintaining documentation transitions are notoriously difficult for libraries even larger than Cypress and we want to make sure that we're able to maintain them properly. So this is why they are on hold.
